src/open_inwoner/conf/app/csp.py

```diff
@@ -0,0 +1,61 @@
+from django.urls import reverse_lazy
+
+from ..utils import config
+
+#
+# Django CSP settings
+#
+# explanation of directives: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+# and how to specify them: https://django-csp.readthedocs.io/en/latest/configuration.html
+#
+# NOTE: make sure values are a tuple or list, and to quote special values like 'self'
+CSP_DEFAULT_SRC = (
+    "'none'",
+)  # ideally we'd use BASE_URI but it'd have to be lazy or cause issues
+CSP_BASE_URI = ("'self'",)
+CSP_FONT_SRC = ("'self'",)
+CSP_SCRIPT_SRC = (
+    "'self'",
+    "'unsafe-eval'",
```

Comment:
ik zie hier nog wel unsafe-eval en unsafe-inline staan, dat is volgens mij niet de bedoeling als ik dit vergelijk met de Django CSP instellingen van openforms: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/conf/base.py